substrate/primitives/trie/src/recorder.rs

```diff
@@ -379,6 +379,19 @@ impl<H: Hasher, I: DerefMut<Target = RecorderInner<H::Out>>> trie_db::TrieRecord
 				// that the value doesn't exist in the trie.
 				self.update_recorded_keys(full_key, RecordedForKey::Value);
 			},
+			TrieAccess::InlineValue { full_key } => {
```

Comment:
@bkchr Would like to double check this with you 🙏 

(introduced by: https://github.com/paritytech/trie/pull/194)